wagtail/wagtailadmin/views/pages.py

```diff
@@ -251,8 +251,13 @@ def create(request, content_type_app_name, content_type_model_name, parent_page_
                     # Ensure the 'next' url is passed through again if present
                     target_url += '?next=%s' % urlquote(next_url)
                 return redirect(target_url)
+
         else:
-            messages.error(request, _("The page could not be created due to validation errors"))
+            if form.non_field_errors:
+                pass
```

Comment:
rather than `pass`, what about:
```
for error in form.non_field_errors:
    messages.error(request, error)
```

Then, you wouldn't need any changes to base.html